README.md

```diff
@@ -217,7 +217,7 @@ Follow these steps if you want to build LLVM yourself. First, download LLVM's
 sources (these instructions use the latest 10.0 release)
```

Comment:
Nit: these instructions no longer use the 'latest' 10.0 release, but rather, 10.0.0 specifically.

Reply:
at time of writing :P